core/resources/scene/config.yaml

```diff
@@ -177,7 +183,10 @@ layers:
         data: {source: osm, layer: roads}
         filter: { name: true, aeroway: false, tunnel: false, railway: false, not: { kind: rail }, $zoom: { min: 10 } }
         draw:
-            text: {}
+            text: 
+                font:
+                    typeface: Regular 15 Roboto
```

Comment:
I agree with @tallytalwar  that breaking the font properties into several would make more sense for native. This would let us use the yaml parsing instead of writing parsing specifically for fonts definition.